types/aws-lambda/trigger/api-gateway-proxy.d.ts

```diff
@@ -66,6 +93,60 @@ export interface APIGatewayProxyResult {
     isBase64Encoded?: boolean;
 }
 
+/**
+ * Works with HTTP API integration Payload Format version 2.0
+ * @see - https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html
+ */
+export interface APIGatewayProxyEventV2 {
+    routeKey: string;
+    rawPath: string;
+    rawQueryString: string;
+    cookies: string[];
+    headers: { [name: string]: string };
+    queryStringParameters: { [name: string]: string };
+    requestContext: {
+      accountId: string;
+      apiId: string;
+      authorizer: { jwt: {
+          claims: { [name: string]: string };
+          scopes: string[]
+          }
+      };
+      domainName: string;
+      domainPrefix: string;
+      http: {
+        method: string;
+        path: string;
+        protocol: string;
+        sourceIp: string;
+        userAgent: string;
+      };
+      requestId: string;
+      routeKey: string;
+      stage: string;
+      time: string;
+      timeEpoch: number;
+    };
+    body: string;
+    pathParameters: { [name: string]: string };
+    isBase64Encoded: boolean;
+    stageVariables: { [name: string]: string };
+    }
```

Comment:
Indentation / wrapping is inconsistent here. I believe there are now rules for prettier.

Reply:
Fixed it. Thanks.